.github/workflows/stale.yml

```
@@ -0,0 +1,19 @@
+name: Mark stale pull requests
+
+on:
+  schedule:
+  - cron: "0 0 * * *"
+
+jobs:
+  stale:
+
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/stale@v3.0.7
+      with:
+        repo-token: ${{ secrets.BEDEVERE_GH_TOKEN }}
```

GitHub Actions has a unique token for every run that has write access to the repo, so reusing Bedevere's token is unnecessary.

```suggestion
        repo-token: ${{ secrets.GITHUB_TOKEN }}
```

---

Ok I've removed the secret